guides/v2.1/frontend-dev-guide/themes/theme-apply.md

```diff
@@ -56,7 +56,7 @@ To add a permanent theme-independent logo:
 2. In the configuration record corresponding to your store view, click **Edit**.
 3. Expand the **Header** tab.
 4. In the **Logo Image** field browse to the logo file saved in your file system.
-6. Upload the file.
+6. Upload the file. Allowed file types: png, gif, jpg, jpeg. to applying .svg logo see the [link]({{page.baseurl}}/frontend-dev-guide/themes/theme-create.html).
```

Comment:
Please, clarify the article's topic link. It's supposed to be `/frontend-dev-guide/themes/theme-create.html#logo_declare`.

Thank you

Reply:
Hello @rogyar I have updated requested changs please review